ProjectSettings/ProjectSettings.asset

```diff
@@ -153,6 +153,7 @@ PlayerSettings:
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
+    Android: com.Vircadia.VircadiaUnitySDK
```

Comment:
This must be a default value for android APK identifier auto generated based on the project and company name, and it's for the example application only. Not sure if it's worth overriding, since it seems like it'll require a whole custom android manifest file, from what I can gather looking around in the settings UI. The actual SDK package identifier that the users will see is "com.vircadia.unitysdk" [here](https://github.com/vircadia/vircadia-unity-sdk/blob/master/Packages/com.vircadia.unitysdk/package.json#L2).